### DISCUSSION TITLE
Text Wrapping

### DISCUSSION THREAD

**Discussion by Digitalhour** [ExperienceLovelace/ha-floorplan#342] [Q&A] [answered]
For some reason I can not get text to wrap. I have tried css and jquery. anyone have any ideas??
 here is my jquery code, If i console.log() it, it wraps in the log, but not on the svg! 
Thoughts? 

                - name: Plex media_summary
                  entities:
                    - entity: media_player.plex_plex_for_lg_lg_oled65c1pub
                      element: media_summary
                  state_action:
                    action: call-service
                    service: floorplan.text_set
                    service_data: ${(entity.attributes.media_summary.replace(/(?![^\n]{1,32}$)([^\n]{1,32})\s/g, '$1\n'))}

**Comment by exetico** [comment]
Please describe what you expect as part of the question.

What's the value of the entity you're trying to manipulate?

What's the value shown in your console?

Does another dumb value works as expected, if that's mapped to the same SVG element?

**Comment by Digitalhour** [comment]
> Please describe what you expect as part of the question.

If I have a long string of text, the text will drop to a new like 

**IE:** "A man, having fallen in love with the wrong woman, is sent by the sultan himself on a diplomatic mission to a distant land as an ambassador. Stopping at a Viking village port to restock on supplies, he finds himself unwittingly embroiled in a quest to banish a mysterious threat in a distant Viking land." 

Using .replace(/(?![^\n]{1,32}$)([^\n]{1,32})\s/g, '$1\n')

**should display as:**
A man, having fallen in love
with the wrong woman, is sent by
the sultan himself on a
diplomatic mission to a distant
land as an ambassador. Stopping
at a Viking village port to
restock on supplies, he finds
himself unwittingly embroiled in
a quest to banish a mysterious
threat in a distant Viking land.
> What's the value of the entity you're trying to manipulate?

I am trying to manipulate the plex attributes, media_summary

> What's the value shown in your console?

A man, having fallen in love
with the wrong woman, is sent by
the sultan himself on a
diplomatic mission to a distant
land as an ambassador. Stopping
at a Viking village port to
restock on supplies, he finds
himself unwittingly embroiled in
a quest to banish a mysterious
threat in a distant Viking land.
> Does another dumb value works as expected, if that's mapped to the same SVG element?

No nothing is wrapping.

**Reply by Digitalhour** [reply, marked as answer]
I updated to the beta 1.0.36 and it works now. sorry should of done that sooner...

**Reply by exetico** [reply]
No problem. It's marked as a pre-release just to maks sure that everything works as expected. I'm happy that it works for you. In release notes, you'll find more details about the update 😊.

**Reply by Digitalhour** [reply]
Thanks @exetico
